client/components/CardContainer.jsx

```diff
@@ -78,6 +82,10 @@ export const CardContainer = () => {
     };
   }, [atThreshold, changeBatch]);
 
+  useEffect(() => {
+    populateCards(questionBatch);
+  }, [questionBatch]);
```

Comment:
i just did this so `populateCards` would run the first time--is this the right way to do that?

Reply:
the first `useEffect`? i wonder then if we should start `changeBatch` as true in the store so it passes the condition on L79? then maybe we don't even need this